c2corg_api/models/feed.py

```diff
@@ -198,6 +205,8 @@ def update_feed_document_create(document, user_id):
     # users can be queried
     DBSession.flush()
 
+    languages = [document.available_langs]
```

Comment:
Isn't `document.available_langs` already a list? And are you sure `document.available_langs` is set at this point? I am wondering if this covered by your tests?